### PR TITLE
[11.0] [MIG] users_ldap_mail

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0

### CODE REVIEW

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

File: users_ldap_mail/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (rgarnau) [inline users_ldap_mail/README.rst]: https

**Comment by pedrobaeza** [comment]
Please move this PR to https://github.com/OCA/server-auth according #1006

**Comment by etobella** [comment]
Migrated to new repository.

**Pull request closed without merging**